website/docs/docs/dbt-cloud-apis/sl-jdbc.md

```diff
@@ -147,9 +147,44 @@ select * from {{
 }}
 ```
 
-</DetailsToggle>
+</Expandable>
+
+<Expandable alt_header="Fetch metrics by substring search">
+
+You can filter your metrics to include only those that contain a specific substring. Use the `search` argument to specify the substring you want to match.
+
+```sql
+select * from {{ semantic_layer.metrics(search='order') }}
+```
+
+If no substring is provided, the query returns all metrics.
 
-<DetailsToggle alt_header="List saved queries">
+</Expandable> 
+
+<Expandable alt_header="Paginate metadata calls">
+
+In cases where user manifests are large, pagination is a useful way to prevent query character limits in the data platform. You can paginate results for both `semantic_layer.metrics()` and `semantic_layer.dimensions()` calls using the `page_size` and `page_number` parameters.
+
+- `page_size`: Optional, sets the number of records per page. If left as `None`, there is no page limit.
+- `page_number`: Optional, specifies the page number to retrieve. Defaults to `1` (first page) if not specified.
```

Comment:
```suggestion
- `page_number`: This is another optional feature which specifies the page number to retrieve. Defaults to `1` (first page) if not specified.
```

Reply:
Hiya @mirnawong1 

I added this non-blocking suggestion as I thought it might flow better as a full sentence.

Kind Regards
Natalie